pkg/azure/sdk_handler.go

```diff
@@ -483,7 +484,7 @@ func (h *AzureSDKHandler) GetSecurityGroup(ctx context.Context, nsgName string)
 	return &nsgResp.SecurityGroup, nil
 }
 
-// GetParagliderVnet returns a valid paraglider vnet, a paraglider vnet is a vnet with a default subnet with the same
+// GetParagliderVnet returns a valid paraglider vnet. A paraglider vnet is a vnet with a default subnet with the same
```

Comment:
It might be worth adding to this that a "paraglider vnet" is one *created* by paraglider.

Reply:
Alright